### PR TITLE
Update 'Quick Links' section of all templates.

### LINKED ISSUE #115: Quick links not working in all pages
This includes adding the links to the footer content.

### LINKED ISSUE #235: Update Quick Links of all templates.
Most of the links in 'Quick Links' section are broken and must be fixed.

### PR DESCRIPTION
**Pull request by dhruvhacks**
In this PR-
Link for JSS Library is replaced with [National Digital Library of India](https://ndl.iitkgp.ac.in/)
Broken links of Mechanical Department and Instrumentation and Control fixed.

@RishabhJain2018 Please review.

fixes #115
closes #235

### CODE REVIEW

File: HTTP_200/templates/about.html [hunk 1]
```diff
@@ -164,7 +164,7 @@ <h4><a href="http://hackncs.com/" target="_blank">Nibble Website</a></h4>
       <div class="row">
         <div class="footer-links">
           <div class="col col-3">
-            <h4>JSS Library</h4>
```
Comment (RishabhJain2018) [inline HTTP_200/templates/about.html]: No no please don't remove this.

File: HTTP_200/templates/about.html [hunk 2]
```diff
@@ -164,7 +164,7 @@ <h4><a href="http://hackncs.com/" target="_blank">Nibble Website</a></h4>
...
+            <h4><a href="https://ndl.iitkgp.ac.in/" target="_blank">National Digital Library of India</a></h4>
```
Comment (RishabhJain2018) [inline HTTP_200/templates/about.html]: use this link here. `http://jssaten.ac.in/library/`

File: HTTP_200/templates/about.html [hunk 3]
```diff
@@ -205,7 +205,7 @@ <h4><a href="http://jssaten.ac.in/Academics/EE/" target="_blank">Electrical Engi
             <h4><a href="http://jssaten.ac.in/Academics/ECE/" target="_blank">Electronics & Communication Engineering</a></h4>
           </div>
           <div class="col col-3">
-            <h4><a href="http://jssaten.ac.in/Academics/IC" target="_blank">Instrumentation and Control</a></h4>
+            <h4><a href="http://jssaten.ac.in/Academics/ice/Index.php" target="_blank">Instrumentation and Control</a></h4>
           </div>
         </div>
       </div>
```
Comment (RishabhJain2018) [inline HTTP_200/templates/about.html]: screen shot 2018-07-08 at 3 24 33 pm
Please replace `My Inbox` from second column with `MBA` and add the link `National Digital Library of India` with hyperlink `https://ndl.iitkgp.ac.in/` below the `JSS Library` by shifting it up.

Please let me know if anything isn't clear to you.

Comment (dhruvhacks) [reply HTTP_200/templates/about.html]: Got it sir! Updating this.

File: HTTP_200/templates/about.html (hunk 2 above)
Comment (dhruvhacks) [reply HTTP_200/templates/about.html]: Okay sir.

File: HTTP_200/templates/about.html (hunk 1 above)
Comment (dhruvhacks) [reply HTTP_200/templates/about.html]: Okay sir. Getting it back.

**Comment by RishabhJain2018** [comment]
@dhruvhacks Can you please paste the screenshot of the footer.

**Comment by RishabhJain2018** [comment]
Yes! you can do that.

**Comment by dhruvhacks** [comment]
@RishabhJain2018 I've updated all the pages. Please review the PR. :slightly_smiling_face:

**Pull request merged**